src/app/server/CommissioningWindowManager.cpp

```diff
@@ -388,6 +383,11 @@ CHIP_ERROR CommissioningWindowManager::StartAdvertisement()
         mWindowStatus = AdministratorCommissioning::CommissioningWindowStatus::kBasicWindowOpen;
     }
 
+    if (mAppDelegate != nullptr)
+    {
+        mAppDelegate->OnCommissioningWindowOpened();
```

Comment:
Can you add a comment in the delegate header file that you could get the type of window opened by calling the right helper method?